artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -3822,6 +3822,30 @@ public void createBridge(final String name,
       }
    }
 
+   @Override
+   public String createBridge(String bridgeConfigurationAsJson) throws Exception {
```

Comment:
This can return `void`. The only reason the `createQueue` JSON method returns a `String` is because the broker applies address settings to the queue on the fly to end up at the final configuration and that's something the client might like to know. In this case the broker just takes what it's given.

Reply:
Done.